third_party/move/move-model/bytecode/abstract_domain_derive/src/lib.rs

```diff
@@ -0,0 +1,136 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+//! Derive macro for `AbstractDomain`
+//!
+//! Currently we can only derive for structs.
+//! For tuple structs, the derived join pair-wise joins each field;
+//! for structs with named fields, the derived join pair-wise joins each field without #[no_join] attribute.
```

Comment:
Update the PR description to refer to `#[no_join]` instead of `#[join]`.

Reply:
done